### PR TITLE
feat: changed content and order of cards in GET Started with API section

### PR DESCRIPTION
**Pull request by utkarsha-deriv**

### CODE REVIEW

**Review by hubert-deriv — DISMISSED** [review]

File: src/features/Home/GetStarted/GetStarted.tsx [hunk 1]
```diff
@@ -9,6 +9,15 @@ export const GetStarted = () => {
           Get started with our API in 3 simple steps:
         </Text>
         <div className={styles.cardContainer}>
+          <a href='/docs/resources/api-guide/' className={styles.mainPageCard} data-testid='guide'>
```
Comment (mohsen-deriv) [inline src/features/Home/GetStarted/GetStarted.tsx:12]: can you please use [Link component](https://docusaurus.io/docs/docusaurus-core#link), thanks :pray:

Comment (mohsen-deriv) [reply src/features/Home/GetStarted/GetStarted.tsx:12]: let me know if you got any problem with it.

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: src/features/Home/GetStarted/GetStarted.tsx (hunk 1 above)
Comment (utkarsha-deriv) [reply src/features/Home/GetStarted/GetStarted.tsx:12]: Hello michio, should I change it for all 3 cards or just this one?

Comment (utkarsha-deriv) [reply src/features/Home/GetStarted/GetStarted.tsx:12]: I changed for all. Let me know if I need to revert the changes back. 😊

Comment (mohsen-deriv) [reply src/features/Home/GetStarted/GetStarted.tsx]: thanks, yeah we have to update the components for all the cards. :pray:

**Comment by mohsen-deriv** [comment]
the build is failing because you have wrong links in the cards, you're using the old project's links, please use the new ones, you can check it out with @hubert-deriv , thanks :pray:

**Review by hubert-deriv — APPROVED** [review]

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**